tests/unit_tests/test_controller_utils.py

```diff
@@ -33,14 +34,9 @@
             'disk_size': 200
         }),
     ])
-def test_get_controller_resources_spot(controller_type,
-                                       custom_controller_resources_config,
-                                       expected, enable_all_clouds,
-                                       monkeypatch):
-    if controller_type == 'spot':
-        controller_resources_config = spot_constants.CONTROLLER_RESOURCES
-    else:
-        controller_resources_config = serve_constants.CONTROLLER_RESOURCES
+def test_get_controller_resources(controller_type,
+                                  custom_controller_resources_config, expected,
+                                  enable_all_clouds, monkeypatch):
```

Comment:
Could we add type hints?

Reply:
Added hint except `enable_all_clouds` and `monkeypatch`. Do we need to add type annotations for those fixture as well?